salt/apps/ocw/templates/webhook-publish.sh.jinja

```diff
@@ -27,6 +27,7 @@ OCW_HUGO_THEMES_GIT_REF={{ ocw_hugo_themes_git_ref }}
 COURSE_BASE_URL={{ course_base_url }}
 OCW_STUDIO_BASE_URL={{ ocw_studio_base_url }}
 OCW_IMPORT_STARTER_SLUG={{ ocw_import_starter_slug }}
+STATIC_API_BASE_URL={{ static_api_base_url }}
```

Comment:
This also needs to be added [here](https://github.com/mitodl/salt-ops/blob/main/salt/apps/ocw/nextgen_build_install.sls#L121) so that it can get passed into this Jinja template.

Reply:
@blarghmatey thanks, should be all set now